macchiato/src/MacchiatoApp.js

```diff
@@ -119,6 +123,7 @@ export default class MacchiatoApp extends Component<any, any> {
 
                     self.nodeId = question.instructions.node._id;
                     self.node = question.instructions.node;
+                    self.prompt = question.instructions.prompt;
```

Comment:
is there a default prompt if there is no prompt set in the instructions object @jtpdowns ?

Reply:
@cabishop at present it will appear as blank (there is no default). I'm also not doing any check for the presence of the `instructions` object, but that's true for other variables here as well.